content/sensu-go/5.19/sensuctl/reference.md

```diff
@@ -648,6 +649,97 @@ See the [RBAC reference][21] for information about using access control with nam
 
 See the [RBAC reference][22] for information about local user management with sensuctl.
 
+## Prune resources
+
+{{% notice important %}}
+**IMPORTANT**: `sensuctl prune` is experimental and under testing for Sensu Go 5.19.0.
+We may remove this feature from 5.19.0 and future releases.
+{{% /notice %}}
+
+**COMMERCIAL FEATURE**: Access `sensuctl prune` in the packaged Sensu Go distribution.
+For more information, see [Get started with commercial features][30].
+
+The `sensuctl prune` command allows you to delete resources that do not appear in your configuration from a file, URL, or STDIN.
+For example, you can use `sensuctl prune` to prune resources that were created by a specific user or that include a specific label selector.
+
+### sensuctl prune usage
+
+{{< highlight shell >}}
+sensuctl prune [RESOURCE TYPE],[RESOURCE TYPE]... [-f [FILE] -f [URL] -f [URL] [-r] ... ] [flags]
+{{< /highlight >}}
+
+In this example `sensuctl prune` command:
+
+- Replace [RESOURCE TYPE] with the [synonym or fully qualified name][48] of the resource you want to prune.
+If you do not specify any resource types, `sensuctl prune` defaults to all types except events and entities.
```

Comment:
If no resource types are provided, the command will actually error. In order to prune all resource types, the `all` keyword must be used.